src/components/RuntimeManager.js

```diff
@@ -128,7 +128,7 @@ export const AppErrorModal = ({ app, onDismiss }) => {
     withErrorReporting('Error loading app details'),
     Utils.withBusyState(setLoadingAppDetails)
   )(async () => {
-    const { errors: appErrors } = await Ajax().Apps.app(app.googleProject, app.appName).details()
+    const { errors: appErrors } = await Ajax().Apps.app(app.cloudContext.cloudResource, app.appName).details()
```

Comment:
It looks like in some places you are confident that `app.cloudContext` will be defined, and in other places you are not confident. What is the difference?

Reply:
The reason we need it is the field comes from an API request and we aren't always going to get 200s

Reply:
I actually think my above comment is incorrect. If we get an app, we are supposed to have a cloudContext and a cloudResource. The reason for the optional chaining was because this whole issue stemmed from a backend bug.